src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-compose-errors.cypress.spec.js

```diff
@@ -19,8 +18,8 @@ describe('Secure Messaging Compose Errors', () => {
     });
 
     PatientComposePage.clickSendMessageButton();
-    // PatientComposePage.verifyErrorText(Data.PLEASE_SELECT_RECIPIENT);
-    // PatientComposePage.verifyFocusOnErrorMessage();
+    PatientComposePage.verifyErrorText(Data.PLEASE_SELECT_RECIPIENT);
+    PatientComposePage.verifyFocusOnErrorMessage();
```

Comment:
@fazilqa I noticed this method validates that focus is on an error field which may be any one of these on the page. How can we validate that focus is on the right error field? I think that may be a more proper way of validating here. I suggest either rewriting this `verifyFocusOnErrorMessage` function where you can probably pass an element as an argument and validate that focus is present or just simply validate focus in the test without a helper function.  Same applies everywhere else in this PR where this function is used

Reply:
done